TEAMS_ROLES_STRUCTURES.md

```diff
@@ -1,38 +1,48 @@
-    - [Golang Core Dev](#golang-ipfs-implementation)
-    - [IPFS GUI](#ipfs-gui)
-    - [IPFS Cluster](#ipfs-cluster)
-    - [IPFS Infrastructure](#ipfs-infrastructure)
-  - **Efforts to Support Specific Uses:**
-    - [Integration with Web Browsers](#integration-with-web-browsers)
-    - [Dynamic Data and Capabilities](#dynamic-data-and-capabilities)
-    - [Decentralized Data Stewardship](#decentralized-data-stewardship)
-    - [Local Offline Collaboration](#local-offline-collaboration-locol)
-  - [**Project**](#project)
-  - [**Community**](#community)
-- [Existing **Research** Groups](#existing-research-groups)
+- [Team Roles](#team-roles)
+  - [IPFS BDFL](#ipfs-bdfl)
+  - [IPFS Project Lead](#ipfs-project-lead)
+  - [IPFS Project Coordinator](#ipfs-project-coordinator)
+  - [Working Group Captain](#working-group-captain)
+  - [Working Group Technical Project Manager](#working-group-technical-project-manager)
+- [Active Teams](#active-teams)
+  - [**Working** Groups](#working-groups)
+    - [**Project**](#project)
+    - [**Community**](#community)
+    - `Development and Maintenance of the Protocol:`
+      - [**JavaScript Core Dev**](#javascript-ipfs-implementation)
+      - [**Golang Core Dev**](#golang-ipfs-implementation)
+      - [**IPFS GUI**](#ipfs-gui)
+      - [**IPFS Cluster**](#ipfs-cluster)
+      - [**IPFS Infrastructure**](#ipfs-infrastructure)
+    - `Efforts to Support Specific Uses:`
+      - [**Integration with Web Browsers**](#integration-with-web-browsers)
+      - [**Dynamic Data and Capabilities**](#dynamic-data-and-capabilities)
+      - [**Decentralized Data Stewardship**](#decentralized-data-stewardship)
+      - [**Local Offline Collaboration**](#local-offline-collaboration-locol)
+  - [**Research** Groups](#research-groups)
 
 ## Description
 
 The IPFS Project is developed by multiple teams from multiple organizations and both intermittent and continuous invidividual contributors. A structure was developed (and continues to evolve) around Working & Research Groups, OKRs, and PM best practices that were designed to support the already large and growing group of people to coordinate among each other at a global scale.
 
-This document captures the Team Structures that emerged and that led to the many of the sucesses within the IPFS project, namely: Working Groups & Research Groups. At the end of this document, you will find pointers to Teams and Projects that have grown to become their own entities, namely: IPLD, libp2p and Multiformats.
+This document presents the the existing teams that emerged and that led to the many of the sucesses within the IPFS project, their structure(namely: Working Groups & Research Groups) and a definition of roles existing in the IPFS org. At the end of this document, you will find pointers to Teams and Projects that have grown to become their own entities, namely: IPLD, libp2p and Multiformats.
 
-#### One line definition
+## Team Structures
```

Comment:
@momack2 probably it is time to describe also what a Tiger Team is given that those have become more and more used.

Reply:
Added!